src/_posts/security/cryptography/2000-01-01-tls.md

```diff
@@ -0,0 +1,24 @@
+---
+title: TLS
+nav: TLS Version
+modified_at: 2023-08-29 00:00:00
+tags: security cryptography tls ciphers
+---
+
+## Definition
+
+Transport Layer Security (TLS) is a protocol that ensures the confidentiality of exchanges between applications and users on the Internet. When a server and client communicate, TLS ensures that no third party can intercept or falsify a message. TLS is the successor to the SSL (Secure Sockets Layer) protocol.
+
+
+This page aims at listing compatible TLS version with the Scalingo platform.
+
+## List of supported versions by our front servers
+
+```
+1.2
+1.3
+```
+
```

Comment:
I find this too long just for stating that we support TLS 1.2 and 1.3.

Plus, it's unclear when this TLS version is implied. It should be made explicit that this is when reaching Scalingo hosted applications.

Shouldn't we also document the list of ciphers we support?

Basically I would take inspiration from the existing page: https://doc.scalingo.com/security/cryptography/ssh